src/components/DSixInput/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+
+const DSixInput = ({ className }) => {
+  const [value, setValue] = useState("");
+
+  const sanitizeInput = (rawInput) => {
+    const currentValue = value;
+    const validValueWhitelist = ["", "1", "2", "3", "4", "5", "6"];
+
+    if (rawInput.length < 0 || rawInput.length > 2) {
```

Comment:
how could a string's length ever be `< 0`?

Reply:
Very good question! 😅 Probably never.